atime_linux_arm.go

```diff
@@ -0,0 +1,14 @@
+// +build arm
```

Comment:
I don't think you need this line? with _linux_arm won't it already only build this on arm?